test/package.json

```diff
@@ -35,7 +33,7 @@
     "webdriver:update": "scripts/webdriver-update"
   },
   "ava": {
-    "snapshotLocation": "test/snapshots",
+    "snapshotDir": "snapshots",
```

Comment:
should this be `snapshots` (as defined here) or `screenshots` (as defined in the .gitignore above)?

Reply:
I'll write this up in the wiki tomorrow and clarify what each suite is and does. A DOM snapshot is not the same as a visreg screenshot.

Reply:
I think a way to make it "unwikiable" is to make it more obvious.

I'm going to change the layout to something like this:

```
test/
  artifacts/
    visregScreenshots/
    domSnapshots/
```

I will include a README.md file in each directory, explaining what each one is used for.

Reply:
Checking this into version control with README.md files is an alright idea, but they get us back to a state where docs require pull requests to update, which is bad for obvious reasons.

I'll make sure README.md files remain short, and point to the wiki for the bulk of the information around the purposes of these directories.